refit for pariah

whataboutism-alos opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

currently changes:
config.txt (pariah!)
admins.txt (removes parent admin list)
motd.txt (pariah!)
silicon_laws (crew member!)

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

updates several files in the config folder to refer to pariah station. note more changes will be needed in the future as this server becomes more than just a vision.

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
config: named server, minor changes to options.
config: named host.
config: replace human being with crew member for DEFAULT silicon laws.
config: replace pre-pariah admin list with just octus for now.
config: motd q.o.l.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

whataboutism-alos committed:
refit for pariah (config)

whataboutism-alos committed:
refit for pariah (admins) (TEMP)

removing long list of existing pre-pariah ckeys, all of which are designated Game Master save the Host.

whataboutism-alos committed:
refit for pariah (motd)

updates motd to clarify server

OctusGit reviewed (changes requested):
Please add the ckey Octus = Host to the list of config admins for now, and change HOSTEDBY Skipjack to HOSTEDBY Octus instead.

whataboutism-alos committed:
refit for pariah (silicon_laws)

changes default law from human being to crew member, considering mostly everyone will be a vox

whataboutism-alos committed:
Update admins.txt

whataboutism-alos committed:
refit for pariah (config) (new)

included changes octus asked for

whataboutism-alos committed:
refit for pariah (whitelist)

removes pre-pariah whitelisted users if ever our own is implemented (sorry fellas no backdoors)
